src/sentry/migrations/0230_auto__del_field_eventmapping_group__del_field_eventmapping_project__ad.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we skip this migration? It does nothing. I assume south was just confused, since nothing ultimately changed.